Integrations/integration-cyberark_AIM.yml

```diff
@@ -191,3 +298,28 @@ script:
   - name: fetch-credentials
     arguments:
     - name: identifier
+  - name: reset-credentials
+    arguments:
+    - name: immediateChangeByCPM
```

Comment:
maybe  a more catchy name. like "change immediately" or John, James, Jim.. ;)

Reply:
I need this name because sometime they return an error saying they need this parameter, so it will be clearer if it has the same name

Reply:
ok.